### ISSUE TITLE
trian error 

### ISSUE THREAD

**Issue by LXWDL** [hujie-frank/SENet#32]
Hi,when i run the train the codes,I have put the corresponding include and cpp files in the caffe file, but still encounter the following error and i don't konw how to solver this problem, can you help me,thanks
] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Axpy (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, Python, ROIPooling, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***

**Comment by hujie-frank** [comment]
It seems that you didn't register the Axpy layer. Did you delete the layer register function 
 call `REGISTER_LAYER_CLASS(Axpy)` at the end of [axpy_layer.cpp](https://github.com/hujie-frank/SENet/blob/master/src/caffe/layers/axpy_layer.cpp)?

**Comment by chunniunai220ml** [comment]
![image](https://user-images.githubusercontent.com/17668109/35895786-fa16f43e-0bf3-11e8-8caf-4eea364f94eb.png)
I do not delete?should i delete?

**Comment by hujie-frank** [comment]
@chunniunai220ml  You should not delete.

**Comment by kyrie0109** [comment]
So could you tell me what's the solution to this problem?

**Comment by xlmou** [comment]
I meet the same problems ,how do you solve it ?
